packages/xod-client/src/debugger/messages.js

```diff
@@ -21,3 +21,5 @@ or if your board is not compatible with XOD runtime code. The original compiler
 message is above. Fix C++ errors to continue. If you believe it is a bug, report the
 problem to XOD developers.`;
 export const TABTEST_RUNNING = 'Running tests...';
+export const TABTEST_ABORTED = 'Tests aborted by user';
```

Comment:
Don't like plural forms here. 
`Running tests...` => `Running test…`
`Tests aborted by user'` => `The test was interrupted by the user`
`Can't get URL to WASM runtime script` => `Unable to get URL of WASM runtime script`